services/QuillLMS/client/app/bundles/Staff/components/comprehension/rulesAnalysis/ruleAnalysis.tsx

```diff
@@ -184,7 +215,7 @@ const RuleAnalysis = ({ history, match }) => {
     { name: "", attribute:"value", width: "750px" }
   ];
 
-  const responseRows = () => {
+  const responseRows = (responses) => {
```

Comment:
Why do we need to pass responses when it's in the local state?

Reply:
because in the hooks at the top, we're passing `ruleFeedbackHistoryData.responses` when there is data and manually setting `setResponses([]);` to initially render an empty table when a time hasn't been selected

Reply:
Gotcha. In that case I think we should change the name of this parameter to avoid conflict.